_topic_maps/_topic_map.yml

```diff
@@ -2560,6 +2560,17 @@ Topics:
     File: logging-5-7-release-notes
 - Name: Support
   File: cluster-logging-support
+- Name: Troubleshooting logging
+  Dir: troubleshooting
+  Topics:
+  - Name: Viewing Logging status
+    File: cluster-logging-cluster-status
+  - Name: Troubleshooting log forwarding
+    File: log-forwarding-troubleshooting
+  - Name: Troubleshooting logging alerts
```

Comment:
```suggestion
  - Name: Troubleshooting Logging alerts
```

Reply:
I think Libby is working on [standardizing the naming conventions and attributes at the moment](https://issues.redhat.com/browse/OBSDOCS-481), so I probably won't make these part of the scope of this PR if that's OK.